pkgs/servers/lenpaste/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitea, git }:
+
+buildGoModule rec {
+  pname = "lenpaste";
+  version = "1.3";
+
+  src = fetchFromGitea {
+    domain = "git.lcomrade.su";
+    owner = "root";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-d+FjfEbInlxUllWIoVLwQRdRWjxBLTpNHYn+oYU3fBc=";
+  };
+
+  buildInputs = [ git ];
```

Comment:
Are you sure this is needed as runtime dependency?
The upstream Dockerfile does not contain `git`.
See https://git.lcomrade.su/root/lenpaste/src/tag/v1.3/Dockerfile

```suggestion
```